From 21fc8a3fca8fbeb0401ae9b11495d579df39cd8f Mon Sep 17 00:00:00 2001
From: John Galt <johngaltfirstrun@gmail.com>
Date: Wed, 7 Dec 2022 14:51:51 -0500
Subject: [PATCH 3/4] ThemePicker: Make ThemesStub a build dependency

This adds integral and expected functionality to ThemePicker, but is a
separate package which must be added during build despite being shipped
in the ThemePicker repo. Insanity.

Simply add as a dependency rather than adding to vendor product
packages.

Change-Id: I57446dca7fd9f3c64ceef4b75e9a510cc213567c
---
 Android.bp | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 9698192c..5520b5b1 100644
--- a/Android.bp
+++ b/Android.bp
@@ -94,7 +94,10 @@ java_defaults {
         ":ThemePicker_src_overrides",
     ],
 
-    required: ["android.software.theme_picker.xml"],
+    required: [
+        "android.software.theme_picker.xml",
+        "ThemesStub"
+    ],
 
     use_embedded_native_libs: true,
 
-- 
2.34.1

